core/sodasql/cli/cli.py

```diff
@@ -31,7 +31,14 @@
 
 LoggingHelper.configure_for_cli()
 logger = logging.getLogger(__name__)
-
+latest_version = ""
+try:
+    latest_version = luddite.get_version_pypi('soda-sql-core')
+except:
+    logger.warning("Cannot check Soda SQL version.")
+if SODA_SQL_VERSION != latest_version:
```

Comment:
Check my comment below, this should't log a warning when we can't determine the latest version.

Reply:
changes done